index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang = "en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+    	<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title>CSS Grid Framework - By Henry & Edward</title>
+		<link rel="stylesheet" href="./dist/css/style.css" />
+		<link rel="stylesheet" href="./theme.css" />
+		 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+		 <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+	</head>
+	<body>
+		<header>
+			<nav style = "display: flex; justify-content: space-between; padding:10px 30px;">
```

Comment:
Do you really need to use inline css in here?? i think its better to use different classes for specific rules in your main css if possible and ignore inline as much as you can.

for example .pd-top50 { padding-top:30px}